src/main/java/stream/flarebot/flarebot/util/GeneralUtils.java

```diff
@@ -567,11 +568,11 @@ public static void handleMultiSelectionCommand(User sender, TextChannel channel,
 
     /**
      * Checks if paths exist in the given json
-     *
+     * <p>
```

Comment:
Start of a p and no end

Reply:
Why was this added I didn't do it xD :eyes: